plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeMetadata.java

```diff
@@ -3113,6 +3113,10 @@ private Optional<SystemTable> getRawSystemTable(ConnectorSession session, Schema
                     systemTableName,
                     getCommitInfoEntries(tableLocation, session),
                     typeManager));
+            case PROPERTIES -> Optional.of(new DeltaLakePropertiesTable(
```

Comment:
Please add the following line to `TestDeltaLakeBasic#testCorruptedTableLocation`
```java
assertQueryFails("SELECT * FROM \"" + tableName + "$properties\"", ".* Table '.*\\$properties' does not exist");
```